include/InputFunctions/sourceTerm.h

```diff
@@ -0,0 +1,29 @@
lipticity_e * inverse_aspect_ratio_epsilon * (r/Rmax) * pow(sin_theta, 2.0) * factor_xi / ( pow((2.0 - temp), 2.0) * temp) + ellipticity_e * inverse_aspect_ratio_epsilon * (r/Rmax) * pow(cos_theta, 2.0) * factor_xi / ( pow((2.0 - temp), 2.0) * temp) + ellipticity_e * cos_theta * factor_xi / ( (2.0 - temp))) + (ellipticity_e * inverse_aspect_ratio_epsilon * (r/Rmax) * sin_theta * cos_theta * factor_xi / ( pow((2.0 - temp), 2.0) * temp) + ellipticity_e * sin_theta * factor_xi / ( (2.0 - temp))) * ((-ellipticity_e) * (inverse_aspect_ratio_epsilon * inverse_aspect_ratio_epsilon) * ((r/Rmax) * (r/Rmax)) * pow(sin_theta, 3.0) * factor_xi / ( pow((2.0 - temp), 2.0) * pow((temp * temp), (3.0 / 2.0))) + 2.0 * ellipticity_e * (inverse_aspect_ratio_epsilon * inverse_aspect_ratio_epsilon) * ((r/Rmax) * (r/Rmax)) * pow(sin_theta, 3.0) * factor_xi / ( pow((2.0 - temp), 3.0) * (temp * temp)) - 3.0 * ellipticity_e * inverse_aspect_ratio_epsilon * (r/Rmax) * sin_theta * cos_theta * factor_xi / ( pow((2.0 - temp), 2.0) * temp) - ellipticity_e * sin_theta * factor_xi / ( (2.0 - temp))) + pow(sin_theta, 2.0) / (temp * temp) - pow(cos_theta, 2.0) / (temp * temp)) / sqrt((-pow((((-ellipticity_e) * inverse_aspect_ratio_epsilon * (r/Rmax) * pow(sin_theta, 2.0) * factor_xi / ( pow((2.0 - temp), 2.0) * temp) + ellipticity_e * cos_theta * factor_xi / ( (2.0 - temp))) * (ellipticity_e * inverse_aspect_ratio_epsilon * (r/Rmax) * sin_theta * cos_theta * factor_xi / ( pow((2.0 - temp), 2.0) * temp) + ellipticity_e * sin_theta * factor_xi / ( (2.0 - temp))) - sin_theta * cos_theta / (temp * temp)), 2.0)) + (pow(((-ellipticity_e) * inverse_aspect_ratio_epsilon * (r/Rmax) * pow(sin_theta, 2.0) * factor_xi / ( pow((2.0 - temp), 2.0) * temp) + ellipticity_e * cos_theta * factor_xi / ( (2.0 - temp))), 2.0) + pow(sin_theta, 2.0) / (temp * temp)) * (pow((ellipticity_e * inverse_aspect_ratio_epsilon * (r/Rmax) * sin_theta * cos_theta * factor_xi / ( pow((2.0 - temp), 2.0) * temp) + ellipticity_e * sin_theta * factor_xi / ( (2.0 - temp))), 2.0) + pow(cos_theta, 2.0) / (temp * temp)))) / sqrt((-pow((((-ellipticity_e) * inverse_aspect_ratio_epsilon * (r/Rmax) * pow(sin_theta, 2.0) * factor_xi / ( pow((2.0 - temp), 2.0) * temp) + ellipticity_e * cos_theta * factor_xi / ( (2.0 - temp))) * (ellipticity_e * inverse_aspect_ratio_epsilon * (r/Rmax) * sin_theta * cos_theta * factor_xi / ( pow((2.0 - temp), 2.0) * temp) + ellipticity_e * sin_theta * factor_xi / ( (2.0 - temp))) - sin_theta * cos_theta / (temp * temp)), 2.0)) + (pow(((-ellipticity_e) * inverse_aspect_ratio_epsilon * (r/Rmax) * pow(sin_theta, 2.0) * factor_xi / ( pow((2.0 - temp), 2.0) * temp) + ellipticity_e * cos_theta * factor_xi / ( (2.0 - temp))), 2.0) + pow(sin_theta, 2.0) / (temp * temp)) * (pow((ellipticity_e * inverse_aspect_ratio_epsilon * (r/Rmax) * sin_theta * cos_theta * factor_xi / ( pow((2.0 - temp), 2.0) * temp) + ellipticity_e * sin_theta * factor_xi / ( (2.0 - temp))), 2.0) + pow(cos_theta, 2.0) / (temp * temp)));
```

Comment:
As runtime polymorphism might not be a possibility, this is one hard-coded test case?